stdlib/zipfile/__init__.pyi

```diff
@@ -94,6 +94,22 @@ class ZipExtFile(io.BufferedIOBase):
 class _Writer(Protocol):
     def write(self, s: str, /) -> object: ...
 
+class _ZipReadable(Protocol):
+    def seek(self, offset: int, whence: int = 0) -> int: ...
```

Comment:
Protocol method arguments should usually be positional-only, unless zipfile actually calls these with keyword arguments.

Reply:
You're right, only positional arguments are used: https://github.com/python/typeshed/pull/12119/commits/97a4a951cba60d162dace2278f62b31fcf6f3f6d

Reply:
Interestingly, MyPy doesn't complain if you try and pass in an implementation that enforces positional arguments, while the protocol doesn't. Pyright seems to have the correct behavior though.

[Pyright Playground
](https://pyright-play.net/?pyrightVersion=1.1.368&strict=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoAK4MYAxmADYCwAULfXY6RQIYDObUASgKYsAmLAEYUeACiJgS5CgEoAXLSjKo-HsCgg%2B-MWx4VgAGigp5mFPgC8UALQBGWbYB8UIXBg82ZgHS%2BGzdk4AQVIYAFcWCl4BYVFFGhVVdU1tXX0jEzNUK1s7YwB6RxsXNw8vJUSVLXCQFFcAInqGZpo1DWAwMDEtGJEeM2jBPqKXADkwFH6oX28Wjq6Q8MjB2PFZWVogA)

[MyPy Playground](https://mypy-play.net/?mypy=latest&python=3.12)

I'll create a MyPy issue on this.